detecto/core.py

```diff
@@ -219,7 +219,7 @@ def __getitem__(self, idx):
 
 class Model:
 
-    def __init__(self, classes=None, device=None, pretrained=True):
+    def __init__(self, classes=None, device=None, pretrained=True, modelname="fasterrcnn_resnet50_fpn"):
```

Comment:
nit: could you replace these strings with single quotes?

also, please add details of this new parameter to the docstring below